tests/lib/cli/rules/fixtures/new-quotes.txt

```diff
@@ -10,21 +10,74 @@ import { debug as d } from '../../utils/debug';
 
 const debug: debug.IDebugger = d(__filename);
 
-// ------------------------------------------------------------------------------
-// Public
-// ------------------------------------------------------------------------------
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
 
 const rule: IRuleBuilder = {
     create(context: RuleContext): IRule {
         // Your code here.
-        const validateFetchStart = (fetchStart: IFetchStart) => {
+        const validateFetchStart = async (fetchStart: IFetchStart) => {
             // Code to validate the rule on the event fetch::start.
+
+            const { resource } = fetchStart;
+            
+            debug(`Validating rule new`);
+            
+            /*
+             * This is where all the magic happens. Any errors found should be
+             * reported using the `context` object. E.g.:
+             * await context.report(resource, null, 'Your error message was here');
+             *
+             * More information on how to develop a rule is available in:
+             * https://sonarwhal.com/docs/developer-guide/rules/index.html
```

Comment:
This should be https://sonarwhal.com/docs/contributor-guide/rules/. The same goes for the links below.